lib/connectors/base/connector.rb

```diff
@@ -19,6 +19,11 @@ def self.display_name
         raise 'Not implemented for this connector'
       end
 
+      # Used as a framework util method, don't override
+      def self.configurable_fields_indifferent_access
+        configurable_fields.with_indifferent_access
+      end
```

Comment:
I think you need to import the dependency for the method: ` 'active_support/core_ext/hash/indifferent_access'`